src/Symfony/Component/Console/Tests/Helper/ProgressBarTest.php

```diff
@@ -307,6 +307,30 @@ public function testRedrawFrequency()
         $bar->advance(1);
     }
 
+    public function testRedrawFrequencyIsAtLeastOneIfZeroGiven()
+    {
+        $bar = $this->getMock('Symfony\Component\Console\Helper\ProgressBar', array('display'),
+            array($output = $this->getOutputStream(), 6));
```

Comment:
and `$output` is not used. please remove the var in testRedrawFrequency and testRedrawFrequencyIsAtLeastOneIfSmallerOneGiven